### DISCUSSION TITLE
Question About Multi-Level Dropdown Menu in Hugo

### DISCUSSION THREAD

**Discussion by Alessio-Siciliano** [imfing/hextra#508] [Q&A] [answered]
Hi there!

I’m new to the Hugo ecosystem and I'm currently using this theme for my new personal blog. I was wondering if it’s possible to implement a multi-level dropdown menu with this theme? I’d like to create a menu with submenus and dropdown functionality for better navigation on my site.

If anyone could point me in the right direction or provide some guidance on how to achieve this, I’d really appreciate it!

Thanks in advance for your help!

**Comment by imfing** [comment, marked as answer]
> I’m new to the Hugo ecosystem and I'm currently using this theme for my new personal blog. I was wondering if it’s possible to implement a multi-level dropdown menu with this theme? I’d like to create a menu with submenus and dropdown functionality for better navigation on my site.

Hi, currently multi-level dropdown is not supported. Please open a feature request and I will implement it in the future.
